packages/sdk/src/transactions/types.ts

```diff
@@ -30,7 +30,7 @@ export type Vout = {
   /**
    * Is spent (Ordit RPC), can be Outpoint `{txid}:{vout}`, or `false` if not spent
    */
-  spent: string | false;
+  spent: boolean;
```

Comment:
hmm, i think ordexer also should return `{txid}:{vout} | false`, is the ordit/client(ordzaar) aware about this changes?

Reply:
On Ordexer, the schema stores it as a boolean: https://github.com/ordzaar/ordexer/blob/29a2424d50e6da02c6d40ef9f2da297a139c1104/packages/db/prisma/schema.prisma#L36

Reply:
So changing it here just makes it consistent across Ordit, Ordexer, and Ordit SDK.

Reply:
need to check whether the client has used the spent string or not, this transaction API will return `string | boolean`
https://github.com/sadoprotocol/ordit/blob/main/src/Utilities/Transaction.ts#L69C45-L69C59